net-misc/aggregate-flim/aggregate-flim-1.0.2.ebuild

```diff
@@ -1,19 +1,17 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Foundation
```

Comment:
If you commit with repoman, it fixes this header for you automatically. You still use old "Foundation".

Reply:
Fixed, and fixed also in my other open pull requests.